src/makeInfinityAdder.js

```diff
@@ -1,10 +1,27 @@
 'use strict';
 
 /***
+ *
  * @return {function}
  */
+
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+
+  const adder = (arg) => {
```

Comment:
Consider using a more descriptive parameter name than `arg`. While it's clear in this context, using a name like `numberToAdd` could make the code more readable.